src/main/scala/zio/config/actions/Read.scala

```diff
@@ -15,7 +14,7 @@ case class Read[A] private (value: ZIO[(Ref[ConfigReport], ConfigSource), List[R
 
 object Read {
   // Read
-  final def read[A](config: Config[A]): Read[A] =
```

Comment:
case FlatMap is not handled

Reply:
I removed it.
In almost all config libs, flatMap is something inherently wrong or monadically not lawful.
For us, we can add it as a separate feature 
It's easy to build a flatMap meanwhile, but after some discussions.